src/StimulusBundle/src/Dto/StimulusAttributes.php

```diff
@@ -129,7 +129,7 @@ public function __toString(): string
         foreach ($this->targets as $key => $targetNamesString) {
             $targetNames = explode(' ', $targetNamesString);
             $targets[$key] = implode(' ', array_map(function (string $targetName): string {
-                return $this->escapeAsHtmlAttr($targetName);
+                return $this->escape($targetName, 'html');
```

Comment:
This could even be simplified as you don't need to escape each of the space-separated values separately (and this will allow avoiding looping first on `$this->targets` and then on `$targets` below.

Side note: to be safe, the handling of targets is missing the escaping of the key

Reply:
Done